Plugin/src/SofaPython3/DataHelper.cpp

```diff
@@ -334,7 +334,10 @@ py::buffer_info toBufferInfo(BaseData& m)
 
     std::tuple<int,int> shape = getShape(&m);
 
-    void* ptr = const_cast<void*>(nfo.getValuePtr(m.getValueVoidPtr()));
+
```

Comment:
// By default, ptr is set to nullptr. It is updated by a call to getValuePtr only if the data is not empty. Otherwise the ptr passed to the py::buffer_info remains nullptr.